react-app/src/components/PatientData.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactJson from 'react-json-view';
+
+function PatientData(props) {
+  return (
+    <div>
+      {props.id < 0 && (
+        <div className="page-text">
+          <p>Select a patient to view their information.</p>
+        </div>
+      )}
+      {props.id >= 0 && (
+        <div className="patient-json-display">
+          <ReactJson src={props.patientJson} collapsed={3} />
+        </div>
+      )}
+    </div>
```

Comment:
I'm assuming the `< 0` is used here to basically check if we have nothing yet since you initialize the state variable to -1 in another file.

I think we should be careful in assuming that ID is a number. For example, if `props.id` is "matt-123", it may still pass this boolean condition, but things get a little dicey.

I'll have a companion comment in the file where the `id` prop is actually initialized and passed along, but basically I think we should be useing `=== null` and `!== null` as our cases here instead of comparing against 0. And then in the other component where the patient ID state variable is initialized we can default it to null (see other comment)